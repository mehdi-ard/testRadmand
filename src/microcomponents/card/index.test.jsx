import React from 'react';
import renderer from 'react-test-renderer';
import Card from '.';

it('renders default', () => {
  expect(
    renderer
      .create(
        <Card title="Default size card" extra={<a href="#">More</a>}>
          <p>test</p>
        </Card>
      )
      .toJSON()
  ).toMatchSnapshot();
});

it('renders hoverable', () => {
  expect(
    renderer
      .create(
        <Card title="Default size card" extra={<a href="#">More</a>}>
          <p>test</p>
        </Card>
      )
      .toJSON()
  ).toMatchSnapshot();
});

it('renders cover', () => {
  expect(
    renderer
      .create(
        <Card title="Default size card" extra={<a href="#">More</a>}>
          <p>test</p>
        </Card>
      )
      .toJSON()
  ).toMatchSnapshot();
});

it('renders bordered', () => {
  expect(
    renderer
      .create(
        <Card
          hoverable
          style={{ width: 240 }}
          cover={
            <img
              alt="example"
              src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"
            />
          }
        >
          <p>test</p>
        </Card>
      )
      .toJSON()
  ).toMatchSnapshot();
});
