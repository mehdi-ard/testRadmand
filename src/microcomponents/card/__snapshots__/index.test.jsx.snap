// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bordered 1`] = `
<div
  className="ant-card ant-card-bordered ant-card-hoverable"
  style={
    Object {
      "width": 240,
    }
  }
>
  <div
    className="ant-card-cover"
  >
    <img
      alt="example"
      src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"
    />
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <p>
      test
    </p>
  </div>
</div>
`;

exports[`renders cover 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Default size card
      </div>
      <div
        className="ant-card-extra"
      >
        <a
          href="#"
        >
          More
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <p>
      test
    </p>
  </div>
</div>
`;

exports[`renders default 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Default size card
      </div>
      <div
        className="ant-card-extra"
      >
        <a
          href="#"
        >
          More
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <p>
      test
    </p>
  </div>
</div>
`;

exports[`renders hoverable 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Default size card
      </div>
      <div
        className="ant-card-extra"
      >
        <a
          href="#"
        >
          More
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <p>
      test
    </p>
  </div>
</div>
`;
